src/styles/styles.js

```diff
@@ -2453,13 +2453,20 @@ const styles = {
         alignItems: 'center',
     },
 
+    checkboxPressable: {
+        borderRadius: 6,
+        padding: 2,
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/27703

The padding here makes the checkbox a few pixels far to the right and does not align with the left line of other inputs. More details https://github.com/Expensify/App/issues/27703#issuecomment-1727995093